tests/Common/MetadataTest.php

```diff
@@ -238,10 +238,10 @@ public function testTableMetadata(string $devBranchType, string $userRole): void
         $this->assertMatchesRegularExpression(self::ISO8601_REGEXP, $metadatas[0]['timestamp']);
         $this->assertEquals(self::TEST_PROVIDER, $metadatas[0]['provider']);
 
-        $mdCopy = $metadatas[0];
-        $mdCopy['value'] = 'newValue';
+        $updateMetadataWith = $testMetadata[0];
+        $updateMetadataWith['value'] = 'newValue';
 
-        $newMetadata = $metadataApi->postTableMetadata($tableId, $provider, [$mdCopy]);
+        $newMetadata = $metadataApi->postTableMetadata($tableId, $provider, [$updateMetadataWith]);
```

Comment:
tohle je relevantni. `postTableMetadata` je form-data, ktery necheckuje co navic je v tom poli metadat. Staci mu, ze tam je key a value. No ona tam je cela response vcetne ID, provider apod. A to se samozrejme nelibi

Reply:
Rovnonu jsi mohl dát explicitně to pole která updatuješ - tj. i key i value. Ale whatever asi :)